src/tir/schedule/primitive.h

```diff
@@ -105,7 +105,7 @@ TVM_DLL std::vector<int64_t> SamplePerfectTile(
  *  The sampled tile size will be partitioned into two parts. The second part has a guarantee
  *  that their extent's product have a factor of `innerpart_factor`. The first part is loops at
  *  [0, partition_pos); the second part is loops at [partition_pos, n) and we will have
- *  `innerpart_factor` | \prod_{l=partition_pos}^{n-1} l.extent
+ *  `innerpart_factor` | prod_{l=partition_pos}^{n-1} l.extent
```

Comment:
Updating this since Clang will warn
```
1 warning generated.
In file included from /home/ruihangl/tvm/src/tir/schedule/primitive/for_kind.cc:19:
In file included from /home/ruihangl/tvm/src/tir/schedule/primitive/../utils.h:49:
/home/ruihangl/tvm/src/tir/schedule/primitive/.././primitive.h:108:26: warning: unknown command tag name [-Wdocumentation-unknown-command]
 *  `innerpart_factor` | \prod_{l=partition_pos}^{n-1} l.extent
                         ^~~~~~
```